pkg/controllers/internalmembercluster/v1beta1/member_controller_integration_test.go

```diff
@@ -138,6 +138,7 @@ var _ = Describe("Test Internal Member Cluster Controller", Serial, func() {
 			By("checking updated member cluster usage")
 			Expect(imc.Status.ResourceUsage.Allocatable).ShouldNot(BeNil())
 			Expect(imc.Status.ResourceUsage.Capacity).ShouldNot(BeNil())
+			Expect(imc.Status.ResourceUsage.Available).ShouldNot(BeNil())
```

Comment:
check that the value is not zero?

Reply:
Hi Ryan! Actually a PR that refactors the tests on the InternalMemberCluster is in progress; would it be OK if this gets addressed in a separate PR? There will be a full check on the total/allocatable/available data.